src/components/sbb-autocomplete/sbb-autocomplete.e2e.ts

```diff
@@ -1,190 +1,183 @@
-import { E2EElement, E2EPage, newE2EPage } from '@stencil/core/testing';
-import events from './sbb-autocomplete.events';
-import optionEvents from '../sbb-option/sbb-option.events';
+import { events } from './sbb-autocomplete';
+import { events as optionEvents } from '../sbb-option';
 import { waitForCondition } from '../../global/testing';
+import { assert, expect, fixture } from '@open-wc/testing';
+import { html } from 'lit/static-html.js';
+import { sendKeys, sendMouse } from '@web/test-runner-commands';
+import { EventSpy } from '../../global/testing/event-spy';
+import { SbbAutocomplete } from './sbb-autocomplete';
+import { SbbFormField } from '../sbb-form-field';
+import '../sbb-option';
 
 describe('sbb-autocomplete', () => {
-  let element: E2EElement, formField: E2EElement, input: E2EElement, page: E2EPage;
+  let element: SbbAutocomplete, formField: SbbFormField, input: HTMLInputElement;
 
   beforeEach(async () => {
-    page = await newE2EPage();
-    await page.setContent(`
+    formField = await fixture(html`
       <sbb-form-field>
         <input />
-        <sbb-autocomplete id="myAutocomplete">
+        <sbb-autocomplete id="myAutocomplete" style="--sbb-options-panel-animation-duration: 0.01s">
```

Comment:
That was the first choice but, for reasons I haven't been able to explain, if I do that the `animationend` event is not fired. After half a day of debugging, I surrendered and used that workaround.
I accepted the compromise since the true meaning of that is just to speed up tests to avoid the timeout (since they have a lot of close/open interactions)